assets/css/style.css

```diff
@@ -79,6 +79,18 @@ h1 {
   justify-content: center;
 }
 
+.game-page-container {
+  background: url('../images/whiskey-bottles.jpg') no-repeat center center fixed;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+  position: relative;
+  align-items: center;
+  display: flex;
+  justify-content: center;

```

Comment:
`height: 100vh` added here will get your new image to cover the whole viewport

Reply:
in next commit